docs/content/users/providers/platform.md

```diff
@@ -42,15 +42,17 @@ web_environment:
 4. Run `ddev pull platform`. After you agree to the prompt, the current upstream databases and files will be downloaded.
 5. Optionally use `ddev push platform` to push local files and database to Platform.sh. The [`ddev push`](../basics/commands.md#push) command can potentially damage your production site, so we don’t recommend using it.
 
-If you have more than one database on your Platform.sh project, you'll need to choose which one you want to use
-as the 'db' primary database on DDEV, and that one will be pulled to the database named `db`.
-Do this by setting PLATFORM_PRIMARY_RELATIONSHIP, for example,
+### Designating a Primary Database
+
+All databases for your Platform.sh project will be pulled locally, with each name matching its remote. When you have more than one, you can optionally tell DDEV which to use as the primary and load into the default `'db'` database.
```

Comment:
Since this won't matter to the vast majority of people, we may want to calm it instead of making it a major header. That's why it was calmed into bullets before; it doesn't matter most of the time, most people have one database. 

```suggestion
If you have just one database on Platform.sh, it will be pulled and loaded into the local default database named 'db'. 

However, if you have more than one database on platform.sh, 
all databases will be pulled locally, with each name matching its remote. If you have more than one, you can optionally tell DDEV which to use as the primary one and load into the default `'db'` database.
```

Reply:
I'm fine with however you want to do it, knowing what you know. Thanks! Sorry this one is so painful.

Reply:
Not at all, it’s an interesting feature to describe simply! I’ll counter-suggest something again shortly.

Reply:
Alright, another stab: https://github.com/drud/ddev/pull/4469/commits/1c0d63ed0d96446ff4cba4525338a2bb939cb15a

I used my theoretical understanding to try and clarify why you _may_ want to designate a primary database, since it’s optional.